### PR TITLE
Add initial suggestions from configuration

### PR DESCRIPTION
**Pull request by mariushoch**
Supersedes #87, fixes https://phabricator.wikimedia.org/T92748

### CODE REVIEW

File: PropertySuggester.php [hunk 1]
```diff
@@ -61,5 +61,10 @@
 	31 // instance of
 );
 
+global $wgPropertySuggesterInitialSuggestions;
+$wgPropertySuggesterInitialSuggestions = array(
+	31, // instance of
+	279 // subclass of
```
Comment (thiemowmde) [inline PropertySuggester.php]: I suggest to keep this base configuration as simple as possible and remove 279 here. Add it to the actual configuration on the Wikimedia cluster.

File: src/PropertySuggester/Suggesters/SuggesterEngine.php [hunk 2]
```diff
@@ -34,10 +34,4 @@ public function suggestByPropertyIds( array $propertyIds, $limit, $minProbabilit
 	 */
 	public function suggestByItem( Item $item, $limit, $minProbability, $context );
 
-	/**
-	 * @param int[] $numericIds - blacklist used to filter suggestions
-	 * @return void
-	 */
-	public function setDeprecatedPropertyIds( array $numericIds );
```
Comment (thiemowmde) [inline src/PropertySuggester/Suggesters/SuggesterEngine.php]: How is this related to this PR?

Comment (mariushoch) [inline src/PropertySuggester/Suggesters/SuggesterEngine.php]: Seems it's not, but it seems ok, so I think we can live with that

File: PropertySuggester.php (hunk 1 above)
Comment (mariushoch) [inline PropertySuggester.php]: Could be done for all configuration variables in a follow-up

**Comment by mariushoch** [comment]
Any travis failures here are just packagist related, can be ignored.
